build

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+MAKE_JOBS="$1"
+COMMIT="$2"
+
+cd /deps
+git clone https://github.com/zig-lang/zig
+cd /deps/zig
+git checkout "$COMMIT"
+mkdir -p /deps/zig/build
+cd /deps/zig/build
+cmake .. -DZIG_STATIC=on -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=/deps/local -DCMAKE_INSTALL_PREFIX=/deps/install
+make "$MAKE_JOBS" install
+/deps/install/bin/zig build --build-file ../build.zig test
```

Comment:
Is the latest up on docker hub? Seeing `/deps/build: line 16: /deps/install/zig: not found`.